catalog/sql/sql_test.go

```diff
@@ -280,7 +280,8 @@ func (s *SqliteCatalogTestSuite) TestCreationOneTableExists() {
 	_, err := sqldb.Exec(`CREATE TABLE "iceberg_tables" (
 		"catalog_name" VARCHAR NOT NULL, 
 		"table_namespace" VARCHAR NOT NULL, 
-		"table_name" VARCHAR NOT NULL, 
+		"table_name" VARCHAR NOT NULL,
+		"iceberg_type" VARCHAR,
```

Comment:
I'd have to look back at the JDBC impl, but should this also be NOT NULL? possibly with a default of TABLE?